apps/rush/src/actions/CheckVersionsAction.ts

```diff
@@ -37,6 +38,11 @@ export default class CheckVersionsAction extends CommandLineAction {
 
     const config: RushConfiguration = RushConfiguration.loadFromDefaultLocation();
 
+    if (config.approvedPackagesPolicyEnabled) {
+      const packageReviewChecker: PackageReviewChecker = new PackageReviewChecker(config);
+      packageReviewChecker.saveCurrentDependencies();
+    }
```

Comment:
@nickpape-msft Fair point.

However, the existing design was very annoying, because after you edit the JSON files, "rush generate" is the only way to read-rewrite them to get nice formatting.  We need a cheap command to do that.

I can move it into an option, like "rush check-versions --update".  I'm open to other suggestions.

---
In reply to: [116122167](https://github.com/Microsoft/web-build-tools/pull/181#discussion_r116122167) [](ancestors = 116122167,116119876)